### ISSUE TITLE
Indirect PP complement throwing validator error:

### ISSUE THREAD

**Issue by bwaldon** [nert-nlp/cgel#143]
https://github.com/nert-nlp/legal-cgel/blob/659939c45cb7269b509072ca6715e5f3584b963b/datasets/oneoff/pdf/id295a8bc5-2a73-11ef-ba6d-a9961dfb011e_0_0.pdf

PP indirect complement "as Congress may deem necessary..." 

The error:

```
line 868, in validate
    assert ch.constituent=='PP' and c_d in {('Nom','Comp'), ('Nom','Comp_ind'), ('VP','Comp'), ('VP','Particle'), ('VP','PredComp'), ('AdjP','Comp'), ...
```

(No error when I remove the PP from the tree).

**Comment by BrettRey** [comment]
It does indeed look like an indirect complement.

**Comment by nschneid** [comment]
In the VP? I guess another alternative would be to say "from time to time" is a supplement within the NP. Or that there's a gap.

**Comment by BrettRey** [comment]
Yes, the supplement analysis seems obvious, now that you mention it.
